app/models/concerns/github_importable.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+module GitHubImportable
```

Comment:
Since this isn't used with an active record object I'm not 100% sure that this could be a model concern.

WDYT?